src/pal/src/memory/heap.cpp

```diff
@@ -151,11 +149,36 @@ HeapCreate(
 #endif // CACHE_HEAP_ZONE
     }
     
+#else // __APPLE__
+    ret = (HANDLE)DUMMY_HEAP;
+#endif // __APPLE__
+
     LOGEXIT("HeapCreate returning HANDLE %p\n", ret);
     PERF_EXIT(HeapCreate);
     return ret;
 }
+
+
+/*++
+Function:
+HeapDestroy
+
+See MSDN doc.
+--*/
+BOOL
+PALAPI
+HeapDestroy(HANDLE hHeap)
```

Comment:
We do not want to be adding non-functional stubbed out implementations to the PAL. If your code is using HeapDestroy, you should add ifdef around it so that it clear that it is not going do what it is supposed to do.